kafka-client/src/test/java/dev/responsive/kafka/async/AsyncProcessorIntegrationTest.java

```diff
@@ -106,6 +106,7 @@
 public class AsyncProcessorIntegrationTest {
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
+  // TODO: consider running a mix of mongo and cassandra for the async tests
```

Comment:
I did run these as Mongo for another check, but we should leave it as Cassandra for the CI/CD for now (or until we address this TODO)